hs/spec_compliance/defs.bzl

```diff
@@ -0,0 +1,36 @@
+"""Create transition to disable hermetic_cc_toolchains for hs targets."""
+
+def _disable_hermetic_cc_transition(_settings, _attr):
+    return {
+        "//bazel:hermetic_cc": False,
+    }
+
+disable_hermetic_cc_transition = transition(
```

Comment:
In f55438e3bd272f3b93bf36e85f9f2e73d77b6dda, I added a patch to `hermetic_cc_toolchain` that adds an extra flag to `target_settings`, and use this transition to opt-out of the hermetic toolchains for the hs build. This is because GHC can't compile with zig, but otherwise the target and platforms are the same.

Is there a better way to do this?

Reply:
excellent question, let me play with this a little

Reply:
Awesome! Worth noting that the [AFL fix](https://github.com/dfinity/ic/pull/3508/commits/19166e3d57cb84b91c273f3a2dc116bc8aeb6ba8) also depends on this now, so whatever we land on won't necessarily go away when the hs builds do.